contact/helpers.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import json
+from urllib.parse import urlparse
+
+import requests
+
+from django.conf import settings
+from django.urls import resolve, Resolver404
+
+
+logger = logging.getLogger(__name__)
+
+INGRESS_URL_SESSION_KEY = 'ingress_url'
+INTERSTITIAL_URL_NAMES = [
+    'contact-us-interstitial-service-agnostic',
+    'contact-us-interstitial-service-specific',
+]
+
+
+def create_zendesk_ticket(cleaned_data, service, ingress_url):
+    auth = requests.auth.HTTPBasicAuth(
+        '{0}/token'.format(settings.CONTACT_ZENDESK_USER),
+        settings.CONTACT_ZENDESK_TOKEN,
+    )
+    data = get_ticket_data(
```

Comment:
I tried using a library instead of baking our own, but that resulted in tickets being created differently than currently - so I've stuck the old format.

Reply:
Add a ticket to Jira for future refactoring @richtier